### PR TITLE
Feature: `withoutHTML`

### PR DESCRIPTION
**Pull request by xMate420x**

### CODE REVIEW

File: packages/withoutHTML/src/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+function removeHTML(ogstr)
+{
+   if ((ogstr===null) || (ogstr===''))
+       return false;
+  else
+   ogstr = ogstr.toString();
+  return ogstr.replace(/<[^>]*>/g, '');
+}
+ 
+const withoutHTML = text => removeHTML(text);
```
Comment (vorillaz) [inline packages/withoutHTML/src/index.js]: Can you please create an inline function? 
```
const withoutHTML = text => {}
```

File: packages/withoutHTML/src/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @description Cleanups a string from HTML tags.
+ * @param {String} text
+ * @example
+ * withoutDiacritics('<b>Test</b>') // returns 'Test'
+ */
+
+function removeHTML(ogstr)
```
Comment (vorillaz) [inline packages/withoutHTML/src/index.js]: This function is redundant, also I think you are not using prettier for linting and the code styling is a bit off.

File: packages/withoutHTML/src/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @description Cleanups a string from HTML tags.
+ * @param {String} text
+ * @example
+ * withoutDiacritics('<b>Test</b>') // returns 'Test'
```
Comment (vorillaz) [inline packages/withoutHTML/src/index.js]: The documentation and the naming is wrong.

File: packages/withoutHTML/test/index.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+it('removes <b> tags from HTML', () => {
+  expect(withoutHTML('<b>Testing</b>')).toBe('Testing');
+});
+
+it('removes multiple tags from HTML', () => {
+  expect(withoutHTML('<b><i>Testing</i></b>')).toBe('Testing');
+});
+
+it('removes tags from a HTML table', () => {
```
Comment (vorillaz) [inline packages/withoutHTML/test/index.js]: Can you please add a few more test cases with HTML attributes?

**Comment by xMate420x** [comment]
Started working on the request, will be done in a few hours

File: packages/withoutHTML/src/index.js (hunk 3 above)
Comment (xMate420x) [reply packages/withoutHTML/src/index.js]: Fixed it

File: packages/withoutHTML/src/index.js (hunk 2 above)
Comment (xMate420x) [reply packages/withoutHTML/src/index.js]: I used the GitHub web editor, so sadly no linting or anything like that. I will lint it and update the files.

File: packages/withoutHTML/src/index.js (hunk 1 above)
Comment (xMate420x) [reply packages/withoutHTML/src/index.js]: Done, hope it's good.

File: packages/withoutHTML/test/index.js (hunk 4 above)
Comment (xMate420x) [reply packages/withoutHTML/test/index.js]: Added, but will add more once I have enough time.

**Comment by vorillaz** [comment]
@xMate420x  You should rebase your branch and update the `packages/plexis` with your new package, also run `yarn test -u` and update the snapshot please.

**Comment by vorillaz** [comment]
closing this in favor of #88

**Pull request closed without merging**